### ISSUE TITLE
cargo install deno failed.

### ISSUE THREAD

**Issue by abcfy2** [denoland/deno#7175]
Here is the output:

```shell
# cargo install deno
# ........
   Compiling dprint-plugin-typescript v0.29.1
error[E0308]: mismatched types
   --> /home/fengyu/.cargo/registry/src/mirrors.ustc.edu.cn-61ef6e0cd06fb9b8/deno-1.3.1/main.rs:554:7
    |
554 |       syntax,
    |       ^^^^^^ expected enum `swc_ecma_parser::Syntax`, found a different enum `swc_ecma_parser::Syntax`
    |
    = note: perhaps two different versions of crate `swc_ecma_parser` are being used?

error[E0308]: mismatched types
   --> /home/fengyu/.cargo/registry/src/mirrors.ustc.edu.cn-61ef6e0cd06fb9b8/deno-1.3.1/main.rs:561:60
    |
561 |       .parse_with_reexports(&module_specifier.to_string(), syntax)
    |                                                            ^^^^^^ expected enum `swc_ecma_parser::Syntax`, found a different enum `swc_ecma_parser::Syntax`
    |
    = note: perhaps two different versions of crate `swc_ecma_parser` are being used?

error[E0308]: mismatched types
   --> /home/fengyu/.cargo/registry/src/mirrors.ustc.edu.cn-61ef6e0cd06fb9b8/deno-1.3.1/lint.rs:173:13
    |
173 |     .syntax(syntax)
    |             ^^^^^^ expected enum `swc_ecma_parser::Syntax`, found a different enum `swc_ecma_parser::Syntax`
    |
    = note: perhaps two different versions of crate `swc_ecma_parser` are being used?

error: aborting due to 3 previous errors

For more information about this error, try `rustc --explain E0308`.
error: failed to compile `deno v1.3.1`, intermediate artifacts can be found at `/tmp/cargo-installgMZvna`

Caused by:
  could not compile `deno`.

To learn more, run the command again with --verbose.
```

**Comment by WangZishi** [comment]
same error here

**Comment by lucacasonato** [comment]
Fixed in #7162. Fix will be released with 1.3.2 (likely this Friday).
